beta/src/pages/learn/your-first-component.md

```diff
@@ -51,58 +51,55 @@ Just like with HTML tags, you can compose, order and nest components to design w
+जैसे-जैसे आपका प्रोजेक्ट बढ़ताी है, आप देखेंगे कि आपके कई डिज़ाइन आपके द्वारा पहले से लिखे गए कौम्पोनॅन्ट का पुन: उपयोग करके, आपके डेवलपमेंट की गति को बढ़ाते हैं। ऊपर दिए गए हमारे टेबल ऑफ़ कंटेंट्स `<TableOfContents />` के साथ किसी भी स्क्रीन पर ऐड किया जा सकता है! आप React ओपन सोर्स कम्युनिटी जैसे [Chakra UI](https://chakra-ui.com/) और [Material UI](https://material-ui.com/) द्वारा शेयर किए गए हजारों कौम्पोनॅन्ट के साथ अपने प्रोजेक्टा को एक शुरुआत दे सकते हैं।
 
-## Defining a component {/*defining-a-component*/}
+## एक कौम्पोनॅन्ट को परिभाषित करना {/* defining-a-component */}
 
-Traditionally when creating web pages, web developers marked up their content and then added interaction by sprinkling on some JavaScript. This worked great when interaction was a nice-to-have on the web. Now it is expected for many sites and all apps. React puts interactivity first while still using the same technology: **a React component is a JavaScript function that you can _sprinkle with markup_**. Here's what that looks like (you can edit the example below):
+परंपरागत रूप से वेब पेज बनाते समय, वेब डेवलपर्स ने अपनी सामग्री को चिह्नित किया और फिर कुछ जावास्क्रिप्ट पर छिड़क कर बातचीत को जोड़ा। यह तब बहुत अच्छा काम करता था जब वेब पर बातचीत एक अच्छी सुविधा थी। अब यह कई साइटों और सभी ऐप्स के लिए अपेक्षित है। एक ही तकनीक का उपयोग करते हुए React पहले अन्तरक्रियाशीलता रखता है: **एक React कौम्पोनॅन्ट एक जावास्क्रिप्ट फ़ंक्शन है जिसे आप _मार्कअप के साथ छिड़क सकते हैं_**। यह कैसा दिखता है (आप नीचे दिए गए उदाहरण को संपादित कर सकते हैं):
 
 <Sandpack>
 
 ```js
 export default function Profile() {
-  return (
-    <img
-      src="https://i.imgur.com/MK3eW3Am.jpg"
-      alt="Katherine Johnson"
-    />
-  )
+  return <img src="https://i.imgur.com/MK3eW3Am.jpg" alt="Katherine Johnson" />;
 }
 ```
 
 ```css
-img { height: 200px; }
+img {
+  height: 200px;
+}
 ```
 
 </Sandpack>
 
-And here's how to build a component:
+यहाँ एक कौम्पोनॅन्ट बनाने का तरीका बताया गया है:
 
-### Step 1: Export the component {/*step-1-export-the-component*/}
+### Step 1: कौम्पोनॅन्ट निर्यात करें {/* step-1-export-the-component */}
 
-The `export default` prefix is a [standard JavaScript syntax](https://developer.mozilla.org/docs/web/javascript/reference/statements/export) (not specific to React). It lets you mark the main function in a file so that you can later import it from other files. (More on importing in [Importing and Exporting Components](/learn/importing-and-exporting-components)!)
+`export default` उपसर्ग एक [standard JavaScript syntax](https://developer.mozilla.org/docs/web/javascript/reference/statements/export) है (React के लिए विशिष्ट नहीं)। यह आपको फ़ाइल में मुख्य फ़ंक्शन को चिह्नित करने देता है ताकि आप बाद में इसे अन्य फ़ाइलों से आयात कर सकें। ([Importing and Exporting Components](/learn/importing-and-exporting-components) में आयात करने पर अधिक!)
```

Comment:
`Importing and Exporting Components` => `इम्पॉर्टिंग एंड एक्सपोर्टिंग कौम्पोनॅन्टस`